<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlavorWindow</class>
 <widget class="QMainWindow" name="FlavorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(221, 244, 182);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="kiosklabel">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>60</y>
      <width>601</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Dyuthi</family>
      <pointsize>100</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(191, 64, 64, 0);</string>
    </property>
    <property name="text">
     <string>Choose !</string>
    </property>
   </widget>
   <widget class="QLabel" name="kiosklabel_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>120</y>
      <width>811</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Dyuthi</family>
      <pointsize>100</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(191, 64, 64, 0);</string>
    </property>
    <property name="text">
     <string>What Flavor ?</string>
    </property>
   </widget>
   <widget class="QLabel" name="turkey">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>310</y>
      <width>251</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Uroob</family>
      <pointsize>56</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icon/free-icon-turkey-12694430.png);
background-color: rgba(191, 64, 64, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="turkey_2">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>280</y>
      <width>250</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Uroob</family>
      <pointsize>56</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(191, 64, 64, 0);
image: url(:/icon/free-icon-turkey-12694430.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="turkey_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>310</y>
      <width>261</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Uroob</family>
      <pointsize>56</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(191, 64, 64, 0);
image: url(:/icon/free-icon-turkey-12694430.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="chocoBtn">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>380</y>
      <width>391</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icon/Screenshot_from_2024-07-04_12-20-15-removebg-preview.png);
background-color: rgba(0, 0, 0, 0); /* 투명 배경 */
    border: none; /* 버튼 테두리 제거 */</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="strawberryBtn">
    <property name="geometry">
     <rect>
      <x>795</x>
      <y>350</y>
      <width>361</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icon/Screenshot_from_2024-07-04_12-19-44-removebg-preview.png);
background-color: rgba(0, 0, 0, 0); /* 투명 배경 */
    border: none; /* 버튼 테두리 제거 */</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="mintBtn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>380</y>
      <width>421</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icon/Screenshot_from_2024-07-04_12-20-02-removebg-preview.png);
background-color: rgba(0, 0, 0, 0); /* 투명 배경 */
    border: none; /* 버튼 테두리 제거 */</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>1920</width>
      <height>820</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-image: url(:/icon/Screenshot from 2024-07-04 10-57-05.png);
    background-position: bottom center; /* 바닥 중앙에서 시작 */
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>kiosklabel</zorder>
   <zorder>kiosklabel_2</zorder>
   <zorder>chocoBtn</zorder>
   <zorder>turkey</zorder>
   <zorder>strawberryBtn</zorder>
   <zorder>turkey_2</zorder>
   <zorder>mintBtn</zorder>
   <zorder>turkey_3</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resourece_order.qrc"/>
 </resources>
 <connections/>
</ui>
